client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -64,10 +65,15 @@ class VerificationCodeForm extends Component {
 	};
 
 	onChangeField = ( event ) => {
+		const {
+			name,
+			value = '',
+		} = event.target;
+
 		this.props.formUpdate();
 
 		this.setState( {
-			[ event.target.name ]: event.target.value,
+			[ name ]: typeof value === 'string' ? value.trim() : value,
```

Comment:
Just curious, when would `value` be something other than a string? Or `undefined`?